Cartopy

jukent opened the pull request:
Instructions for the tutorial on Wednesday led by @michaelavs

- Maybe needs a first Cartopy command to test the install (and because cartopy downloads some things at first run that it doesn't do at install)

michaelavs commented on site/posts/cartopy-tutorial.ipynb:
```diff
@@ -0,0 +1,61 @@
+    "\n",
+    "<!-- TEASER_END -->\n",
+    "\n",
+    "- Navigate to Michaelsa's [Cartopy Tutorial Github Repository](https://github.com/michaelavs/cartopy_tutorial).\n",
```
```suggestion
    "- Navigate to Michaela's [Cartopy Tutorial Github Repository](https://github.com/michaelavs/cartopy_tutorial).\n",
```

jukent replied:
Sorry for the typo!

michaelavs commented on site/posts/cartopy-tutorial.ipynb:
```diff
@@ -0,0 +1,61 @@
+    "\n",
+    "- In your terminal, type `git clone PASTE` where you paste the SSH address you just copied (`git clone git@github.com:michaelavs/cartopy_tutorial.git`)\n",
+    "\n",
+    "- Create a Conda environment from the `conda_environment.yaml` file with the terminal command `conda env create -f conda_environment.yml`. Make sure you are in the local `cartopy_tutorial` directory for this step.\n",
```
```suggestion
    "- Create a Conda environment from the `conda_environment.yml` file with the terminal command `conda env create -f conda_environment.yml`. Make sure you are in the local `cartopy_tutorial` directory for this step.\n",
```

michaelavs commented on site/posts/cartopy-tutorial.ipynb:
```diff
@@ -0,0 +1,61 @@
+    "\n",
+    "- Test your installation in the terminal with `conda activate cartopy_tutorial`. \n",
+    "\n",
+    "If you did not attend the live session, it will beneficial to watch the recording from the live tutorial on [Jupyter Notebooks](https://youtu.be/xSzXvwzFsDU) and on [Matplotlib](https://youtu.be/EiPRIdHQEmE) before the Cartopy session.\n",
```
```suggestion
    "If you did not attend the live session, it will be beneficial to watch the recording from the live tutorial on [Jupyter Notebooks](https://youtu.be/xSzXvwzFsDU) and on [Matplotlib](https://youtu.be/EiPRIdHQEmE) before the Cartopy session.\n",
```

michaelavs commented:
I just added "test_cartopy_install" to the cartopy repo. It's very basic, but all it does is read in the packages that should be installed with their conda environment and then makes a basic cartopy projection using NaturalEarthFeature, cfeature, and adding coastlines. I also added comments to help troubleshoot if they do encounter any errors with running things. Let me know if you want me to add or change anything with that file!

Past that, I only found those few spelling errors (which it looks like you've already addressed), so this looks good to me!

jukent commented:
> I just added "test_cartopy_install" to the cartopy repo. It's very basic, but all it does is read in the packages that should be installed with their conda environment and then makes a basic cartopy projection using NaturalEarthFeature, cfeature, and adding coastlines. I also added comments to help troubleshoot if they do encounter any errors with running things. Let me know if you want me to add or change anything with that file!
> 
> Past that, I only found those few spelling errors (which it looks like you've already addressed), so this looks good to me!

Thanks Michaela!
